Client/Application/Delegates/SceneDelegate.swift

```diff
@@ -15,6 +15,7 @@ private let log = Logger.browserLogger
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
+    internal var window: UIWindow?
```

Comment:
Why do we need this internal window here ?

Reply:
Chromium will Crash without it.

Reply:
Why is Chromium crashing without it? This seems like a very weird hack... if absolutely necessary then needs comments around it why

Reply:
Chromium doesn't realize that `Window` is optional in `SceneDelegate`. They legit `force unwrap` it in their own implementation and they always expect this property to exist and to be non-null.

I don't think it really needs any docs because we just set the variable anyway like we used to do in `AppDelegate`. It's just that it's optional and Chromium doesn't care about that. It uses it as the `main` window even though you can have `multiple` windows lol.